Add -pie -fPIE flags for gol to fix I:hardening-no-pie 
--- a/Makefile.am
+++ b/Makefile.am
@@ -5,7 +5,7 @@
 
 bin_PROGRAMS = gol
 gol_SOURCES = gol.c gol.h compatibility.h
-gol_CFLAGS = $(GTHREAD2_CFLAGS) $(GTK2_CFLAGS) $(OPENSSL_CFLAGS) $(SQLITE3_CFLAGS) $(APP_INDICATOR_CFLAGS) -DDATADIR='"$(pkgdatadir)"' -DLIBDIR='"$(pkglibdir)"'
+gol_CFLAGS = $(GTHREAD2_CFLAGS) $(GTK2_CFLAGS) $(OPENSSL_CFLAGS) $(SQLITE3_CFLAGS) $(APP_INDICATOR_CFLAGS) -DDATADIR='"$(pkgdatadir)"' -DLIBDIR='"$(pkglibdir)"' -pie -fPIE
 gol_LDADD = $(GTHREAD2_LIBS) $(GTK2_LIBS) $(OPENSSL_LIBS) $(SQLITE3_LIBS) $(GMODULE2_LIBS) $(APP_INDICATOR_LIBS)
 
 EXTRA_DIST = gol.rc Makefile.w32 README.mkd TODO data/gol.desktop VERSION
